UNRELEASED.md

```diff
@@ -2,6 +2,8 @@
 
 ### Breaking changes
 
+- Update references to border color variants ([#2811](https://github.com/Shopify/polaris-react/pull/2811))
```

Comment:
Let’s just remove this entry because we are not considering any of these changes breaking until after we default to the new design language and add a "skip changelog" label